server/model/reviewAndSubmitAdaViewModel.ts

```diff
@@ -69,4 +69,36 @@ export default class ReviewAndSubmitAdaViewModel {
     }
     return null
   }
+
+  public removeTable() {
+    if (!this.quashedTable() && !this.createTable()) {
+      return {
+        caption: 'ADA details',
+        head: [
+          { text: 'Last updated' },
+          { text: 'From date' },
+          { text: 'To date' },
+          { text: 'Days', format: 'numeric' },
+        ],
+        rows: [
+          ...this.existingAdjustments.map(it => {
+            return [
+              { text: dayjs(it.lastUpdatedDate).format('D MMM YYYY') },
```

Comment:
have you got a global date format for adjustments? Are all dates consistently formatted?